### DISCUSSION TITLE
How does linkerd-proxy's ingress mode work?

### DISCUSSION THREAD

**Discussion by NikitaSkrynnik** [linkerd/linkerd2#10403] [Help] [answered]
Hello. Could anybody tell me how linkerd-proxy's `ingress mode` works, please? 

I have two questions:
1. It looks like `ingress mode` checks `l5d-dst-override` header when it gets HTTP request. Who sets this header?
2. Can linkerd's outbound proxy also check other headers? For example, `Host` header in HTTP request?

**Comment by wmorgan** [comment, marked as answer]
You should set that header, i.e. in the ingress config. Please see https://linkerd.io/2.12/tasks/using-ingress/#ingress-details and the various examples above making use of it.
